index.html

```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nunito:wght@200&family=Roboto:ital,wght@0,100;0,300;0,400;0,700;0,900;1,700&display=swap"
+      rel="stylesheet"/>
+
+    <title>Home</title>
+
+  </head>
+  <body class="font-nunito">
+    <div class="header">
+      <div class="navbar">
+        <div class="navbar__container">
+          <div class="navbar__logo">
+            <img class="navbar__image" src="images/Vector.png " alt="" />
+          </div>
+          <ul class="navbar__lists">
+            <li class="navbar__menu">
+              <a href="#" class="navbar__links">Home</a>
+            </li>
+            <li class="navbar__menu">
+              <a href="#" class="navbar__links">Product</a>
+            </li>
+            <li class="navbar__menu">
+              <a href="#" class="navbar__links">Pricing</a>
+            </li>
+            <li class="navbar__menu">
+              <a href="About Us.html" class="navbar__links">About Us</a>
+            </li>
+            <li class="navbar__menu">
+              <a href="#" class="navbar__links">Blog</a>
+            </li>
+            <li class="navbar__menu">
+              <a href="Contact.html" class="navbar__links">Contact</a>
+            </li>
+            <li class="navbar__menu navbar__menu--button">
+              <a href="#" class="navbar__links">Free trial</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="content">
+        <div class="content__background"></div>
+        <div class="content__container">
+          
+          <h1 class="content__title">
```

Comment:
you can have content__container-title to make it more understandable as the title is the content in the container